### PR TITLE
Feature/schedule test provider

### PR DESCRIPTION
**Pull request by AlviseSembenico**

### CODE REVIEW

File: src/vng/servervalidation/task.py [hunk 1]
```diff
@@ -28,6 +28,13 @@ def get_jwt(server_run):
     )
 
 
+@app.task
+def execute_test_scheduled():
```
Comment (alextreme) [inline src/vng/servervalidation/task.py]: @AlviseSembenico does the user have the ability to cancel the scheduled ServerRun's?

**Comment by alextreme** [comment]
@AlviseSembenico also check the tests, seems there's one failing:

http://jenkins.maykin.nl/job/VNG Github/job/api-testvoorziening/view/change-requests/job/PR-119

File: src/vng/servervalidation/task.py (hunk 1 above)
Comment (AlviseSembenico) [reply src/vng/servervalidation/task.py]: True, I am working on it

**Comment by AlviseSembenico** [comment]
@alextreme Fixed the issued, added the tests and the possibility to stop them

File: deployment/vars/test.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ pub_key_users:
   - joeri
 
 project_repo: git@github.com:VNG-Realisatie/api-testvoorziening.git
-branch: develop
+branch: feature/schedule-test-provider
```
Comment (alextreme) [inline deployment/vars/test.yml]: @AlviseSembenico I don't think you want to commit this change

File: src/celerybeat.pid [hunk 3]
```diff
@@ -0,0 +1 @@
+6414
```
Comment (alextreme) [inline src/celerybeat.pid]: @AlviseSembenico also put this in your .gitignore

Comment (alextreme) [reply src/celerybeat.pid]: and also remember to git rm this filem, and dump.rdb above as well :)

File: src/vng/servervalidation/api_views.py [hunk 4]
```diff
@@ -66,6 +66,34 @@ def perform_create(self, serializer):
         else:
             server = serializer.save(user=self.request.user, pk=None, started=timezone.now())
 
+# TODO: decorate it setting the response serializer
```
Comment (alextreme) [inline src/vng/servervalidation/api_views.py]: @AlviseSembenico do you still need to do this?

**Comment by AlviseSembenico** [comment]
@alextreme Could you please check it out?

**Comment by alextreme** [comment]
@AlviseSembenico as discussed as an end-user I'd expect the ability to 'Run now' such a scheduled run, and see via the WebUI that the run is in the 'running' state.

**Comment by alextreme** [comment]
And ensure the tests are passing again: http://jenkins.maykin.nl/job/VNG Github/job/api-testvoorziening/view/change-requests/job/PR-119

**Pull request merged**